<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeFormClass</class>
 <widget class="QWidget" name="EmployeeFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeForm</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">

QTabWidget::pane { background: transparent; border: none;}</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normaloff>../icons/general.png</normaloff>../icons/general.png</iconset>
    </attribute>
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>109</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>74</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelId">
        <property name="text">
         <string>ID (*)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="firstNameLabel">
        <property name="text">
         <string>First name (*)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="firstNameEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lastNameLabel">
        <property name="text">
         <string>Last name (*)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lastNameEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="passwordCheckbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow to use password</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>531</width>
       <height>195</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>52</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="confirmPasswordLabel">
        <property name="text">
         <string>Cofirm password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="confirmPasswordEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="departmentLabel">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="departmentCombobox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dateOfBirthLabel">
        <property name="text">
         <string>Date of birth</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateOfBirthEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="workingDateLabel">
        <property name="text">
         <string>Start working date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="workingDateEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="disableEmployeeCheckbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Employee is disabled</string>
     </property>
    </widget>
    <widget class="QLabel" name="photoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>34</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Photo</string>
     </property>
    </widget>
    <widget class="QLabel" name="avatarLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/no-image.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBrowse">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearPhoto">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>420</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
     </property>
     <property name="text">
      <string>Clear photo</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="pathLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>440</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>../icons/contact.png</normaloff>../icons/contact.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Contact Information</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="emailEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="phoneLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="phoneEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="cellPhoneEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cellPhoneLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Cell phone</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="addressLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>94</height>
         </size>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="addressTextEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="icon">
     <iconset>
      <normaloff>../icons/eye.png</normaloff>../icons/eye.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Irises</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>541</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="noticeLeftLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: red;
}</string>
        </property>
        <property name="text">
         <string>(? Not register yet )</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Right iris</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="noticeRightLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: red;
}</string>
        </property>
        <property name="text">
         <string>(? Not register yet )</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Left iris</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="rightIrisImgLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>251</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/no-iris.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="leftIrisImgLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>251</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../icons/no-iris.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>560</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/check.png</normaloff>../icons/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>560</y>
     <width>95</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	text-align: center;
	color: #fff;
	background-color: rgba(70, 133, 210, 1);
	border-radius: 4px;
}

QPushButton:hover {
	background-color: rgba(70, 133, 210, 0.7);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/close.png</normaloff>../icons/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>12</width>
     <height>12</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>600</y>
     <width>588</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
